MonoGame.Framework/Game.cs

```diff
@@ -19,6 +19,10 @@ namespace Microsoft.Xna.Framework
 {
     public class Game : IDisposable
     {
+#if DESKTOPGL
+        public static bool Resizable = false;

```

Comment:
> We will have to find a way around that. At least on startup we could delay creation of the SDL window until Game.Initialize() which currently when the window is actually shown.
> 
> But changing that after startup... including making the window borderless... is all stuff we need to support.
> 
> Regardless of all that we won't be adding a Game.Resizable property.

Window creation is tied to a valid GL window in: https://github.com/cra0zy/MonoGame/blob/sdl2/MonoGame.Framework/Graphics/GraphicsDevice.OpenGL.cs#L100 I though doing this was more sensible for the moment.

Window can already be set borderless without problem, it's only Resizable property that is not setable after creation.